src/wallet/parser.rs

```diff
@@ -19,6 +19,22 @@ pub enum OrdParser {
 }
 
 impl OrdParser {
+    /// Unwraps the [Brc20] from the [OrdParser] enum.
+    pub fn unwrap_brc20(self) -> Brc20 {
```

Comment:
@veeso There are TryFrom implementations for the OrdParser that successfully do these conversions. There examples in the tests, so I'm not sure about this.